client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -587,26 +594,30 @@ export class PluginsList extends Component {
 	};
 
 	maybeShowDisconnectNotice() {
-		const { translate } = this.props;
-
 		if ( this.state.disconnectJetpackNotice && ! this.props.inProgressStatuses.length ) {
 			this.setState( {
 				disconnectJetpackNotice: false,
 			} );
 
-			this.props.warningNotice(
-				translate(
-					'Jetpack cannot be deactivated from WordPress.com. {{link}}Manage connection{{/link}}',
-					{
-						components: {
-							link: <a href={ '/settings/manage-connection/' + this.props.selectedSiteSlug } />,
-						},
-					}
-				)
-			);
+			this.showDisconnectionNotice();
 		}
 	}
 
+	showDisconnectionNotice() {
+		const { translate } = this.props;
+
+		this.props.warningNotice(
+			translate(
+				'Jetpack cannot be deactivated from WordPress.com. {{link}}Manage connection{{/link}}',
+				{
+					components: {
+						link: <a href={ '/settings/manage-connection/' + this.props.selectedSiteSlug } />,
```

Comment:
Can we move `this.props.selectedSiteSlug` above to the `this.props` destructuring? And it's better to write this as 

```
`/settings/manage-connection/${this.props.selectedSiteSlug}`
```